pkg/model/database/database.go

```diff
@@ -135,6 +135,7 @@ type Project struct {
 	BuildDefinition string    `gorm:"not null;default:''"`
 	Branches        []Branch  `gorm:"foreignKey:ProjectID;constraint:OnUpdate:CASCADE,OnDelete:CASCADE"`
 	GitURL          string    `gorm:"not null;default:''"`
+	RemoteProjectID string    `gorm:"not null;default:''"`
```

Comment:
nit: Suggest placing this just below the ProjectID field. I think it belongs better next to that one